docker/local-universe/Dockerfile.static

```diff
@@ -0,0 +1,30 @@
+## The versions in this file are hardcoded for audit purposes, and should not be changed.
+FROM registry:2.4.1
+
+ENV NGINX_VERSION 1.12.0-1~jessie
+ENV NJS_VERSION 1.12.0.0.1.10-1~jessie
+ENV GETTEXT_VERSION 0.19.3-2
+
+RUN apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62 \
```

Comment:
The one in Dockerfile.base is unreliable (it occasionally doesn't work).  This pulls from the pool of keyservers rather than directly from pgp.mit.edu:80.